packages/unified/src/remark/utils.ts

```diff
@@ -365,6 +366,45 @@ const getLinkCandidates = async ({
 };
 
 export class LinkUtils {
+  static isEquivalent(linkA: DLink, linkB: DLink): boolean {
```

Comment:
These methods seem more appropriate somewhere in `common-all`.

Same can be said about a lot of methods in this class... we should break down this file into methods that need to be in `unified` package and those that don't. Maybe in the next iteration of refactoring.

Reply:
Makes sense. I moved `BacklinksUtil` into common-all for this reason. Let's do it in another step of refactoring